hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RocksDatabase.java

```diff
@@ -1001,4 +1003,8 @@ protected void finalize() throws Throwable {
     }
     super.finalize();
   }
+
+  public ManagedRocksDB getDb() {
```

Comment:
nit: `getManagedRocksDB()` because `OmSnapshot.getMetadataManager().getStore()).getDb().getDb()` is confusing.
I agree that variable name could have been better. But it is what it is now.

Reply:
done